app/assets/stylesheets/game_day.css.scss

```diff
@@ -0,0 +1,21 @@
+body#game_day {
+  &:after{
+    content: "beta";
```

Comment:
Properties should be ordered background, color, content, font-family, font-size, font-weight, height, left, line-height, position, text-align, text-transform, top, -ms-transform, -webkit-transform, transform, width